scripts/setup_benchmark.py

```
@@ -0,0 +1,49 @@
+
+    # Insert 50 crew members
+    crew_list = []
+    for i in range(50):
+        crew_list.append(
+            {
+                "name": f"Crew Member {i}",
+                "role": ["WC", "CM", "CC", "FFC", "FFD", "FFT"][i % 6],
+                "skills": "TTR LGV BA" if i % 2 == 0 else "BA",
+                "contract_hours": "42.0 hours",
+            }
+        )
```

Ruff (0.15.5)

[warning] Use a list comprehension to create a transformed list

(PERF401)